confgenerator/testdata/invalid/linux/traces-wrong_receiver_type/golden/error

```diff
@@ -0,0 +1 @@
+traces receiver "otlp" from pipeline "otlp" is not defined.
```

Comment:
Can we make this consistent with logging and metrics sides? Would that require a lot of efforts? If so, ignore this.

aka adjust the folder name to `traces-receiver_unsupported_type` and adjust the error message to:

```
traces receiver with type "apache" is not supported. Supported logging receiver types: [otlp].
```


Reference:
https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/testdata/invalid/linux/logging-receiver_unsupported_type/golden/error

has

```
logging receiver with type "unsupported_type" is not supported. Supported logging receiver types: [apache_access, apache_error, cassandra_debug, cassandra_gc, cassandra_system, couchbase_general, couchbase_goxdcr, couchbase_http_access, couchdb, elasticsearch_gc, elasticsearch_json, files, flink, fluent_forward, hadoop, hbase_system, jetty_access, kafka, mongodb, mysql_error, mysql_general, mysql_slow, nginx_access, nginx_error, oracledb_alert, oracledb_audit, postgresql_general, rabbitmq, redis, saphana, solr_system, syslog, systemd_journald, tcp, tomcat_access, tomcat_system, varnish, vault_audit, wildfly_system, zookeeper_general].
```

Reply:
It's not testing the same thing as `receiver_unsupported_type`; this is testing trying to reference a metrics receiver in a trace pipeline, not trying to use a metrics receiver type in a trace receiver definition. I just renamed the receiver in the test to make the golden error a little clearer.